internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -77,11 +83,29 @@ func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOp
 			Variables:    config.Variables,
 			MountPoints:  mp,
 			DockerLabels: config.DockerLabels,
+			DependsOn:    config.DependsOn,
 		})
 	}
 	return sidecars, nil
 }
 
+// convertDependsOn converts an Image DependsOn field to a template DependsOn version
+func convertImageDependsOn(s convertSidecarOpts) (map[string]string, error) {
+	if s.imageConfig == nil || s.imageConfig.DependsOn == nil {
+		return nil, nil
+	}
+	if err := validateImageDependsOn(s); err != nil {
+		return nil, err
+	}
+	return s.imageConfig.DependsOn, nil
+}
+
+type convertSidecarOpts struct {
```

Comment:
Can we move this struct up to just before the definition of `convertSidecar` for readability?

Reply:
Moved.